onnx/defs/tensor_proto_util.cc

```diff
@@ -27,45 +27,47 @@ namespace ONNX_NAMESPACE {
     return t;                                                   \
   }
 
-  inline bool is_platform_little_endian() {
-    int num = 1;
-    if (*(char*)&num == 1)
-      return true;
-    return false;
-  }
+inline bool is_platform_little_endian() {
+  int num = 1;
+  if (*(char*)&num == 1)
+    return true;
+  return false;
+}
 
-#define DEFINE_PARSE_RAW_DATA(type)                                        \
+#define DEFINE_PARSE_DATA(type, typed_data_fetch)                          \
```

Comment:
Nice refactor. We can unify the data parsing step after this! Thanks.

btw for testing purpose, did you have the model modified (pass the tensor as initializor instead of input) and tested with the changed code? If you need I can send you the script to do so offline.

Reply:
Thanks @raymondxyang. Not sure if this will cover the case - but I added a shape inference test for upsample that has an initializer with raw data. Will this help ?

As an aside  - I see that the test seems to be for eopset 9, should I add similar tests for opset 10 ?

Reply:
Never mind - I see that the op is deprecated in opset 10. Please just check if he test I added will cover your case. Thanks!

Reply:
Looks it will cover the case. Thanks!